gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -820,6 +820,105 @@ def _create_node_config_data(self, auth_token, tenant_id,
                     'description'] = str(common_desc)
 
                 nf_desc = str(firewall_desc)
+        elif service_type == pconst.VPN:
+            # rvpn_l3_policy = self._get_rvpn_l3_policy(auth_token,
+            #    provider, update)
+            # if rvpn_l3_policy is None:
+            #    return None, None
+            # config_param_values['ClientAddressPoolCidr'] = rvpn_l3_policy[
+            #    'ip_pool']
+            config_param_values['Subnet'] = (
+                consumer_port['fixed_ips'][0]['subnet_id']
+                if consumer_port else None)
+            l2p = self.gbp_client.get_l2_policy(
```

Comment:
Can we check if l2p and l3p are already retrieved to reuse them. If not, then leave this code as is.